field/goldilocks/goldilocks.go

```diff
@@ -140,7 +145,7 @@ func NegOne() *Element {
 }
 
 func Sample() Element {
-	elem := g.NewElement(0)
+	elem := G_ZERO
 	elem.SetRandom()
```

Comment:
I think this mutates the `elem`. We should not return mutable data from other constant functions too like `Zero()` etc